cami.c

```diff
@@ -101,14 +101,17 @@ struct ami_session {
 	unsigned int return_null_on_error:1;
 };
 
+static int ami_debug_level = 0;
+
 static struct ami_session *ami_session_new(void)
 {
 	struct ami_session *ami = calloc(1, sizeof(*ami));
 	if (!ami) {
 		return NULL;
 	}
 	ami->ami_socket = -1;
-	ami->debugfd = -1;
+	ami->debugfd = STDERR_FILENO;
```

Comment:
This is intentionally `-1` since this library is mostly used in daemon programs that wouldn't want stuff logged to the console by default. `ami_set_debug()` can be used to set the logging fd.